.github/scripts/utils.py/check-services.py

```diff
@@ -313,9 +278,11 @@ def main():
         if not new_service["servers"]:
             print(f'💀 Service "{service["name"]}" has no valid servers left, removing!')
             affected_services[service["name"]] = f"Service removed"
-            continue
+        else:
+            new_services["services"].append(new_service)
 
-        new_services["services"].append(new_service)
+        # wait a bit between services
+        time.sleep(2.0)
```

Comment:
```suggestion
        await asyncio.sleep(2)
```

Using a sync wait in a coroutine is probably bad for the event loop.